environment.yml

```diff
@@ -5,6 +5,6 @@ channels:
 dependencies:
   - python=3.7
   - jax=0.3.14
-  - jaxlib=0.3.14
+  - jaxlib=0.3.14=cpu*
```

Comment:
just checking that this should be `=cpu*` and not `=*cpu*`, if the former works that's probably preferable?

Reply:
I suggested the change from `=*cpu*` to `=cpu*`, but only had a very minor preference (`cpu` is at the beginning of the string in the current schema; I think it comes down to whether we want things to break if they change the schema or continue to work (but slight chance of doing something unexpected))